src/mscorlib/src/Microsoft/Win32/Win32Native.cs

```diff
@@ -701,14 +701,15 @@ internal unsafe struct MEMORY_BASIC_INFORMATION {
         internal const String OLE32    = "ole32.dll";
         internal const String OLEAUT32 = "oleaut32.dll";
         internal const String NTDLL    = "ntdll.dll";
+        internal const String ADVAPI32 = "advapi32.dll";
```

Comment:
It's for GetUserName, and you're right that it's currently desktop-only.  Hopefully we could get that changed.